packages/tyxml-jsx/tyxml-jsx.4.5.0/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+synopsis: "JSX syntax to write TyXML documents"
+description: """
+```reason
+open Tyxml;
+let to_reason = <a href="reasonml.github.io/"> "Reason!" </a>
+```
+ 
+The TyXML JSX allow to write TyXML documents with reason's JSX syntax. 
+It works with textual trees, virtual DOM trees, or any TyXML module.
+"""
+maintainer: ["dev@ocsigen.org"]
+authors: ["The ocsigen team"]
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+homepage: "https://github.com/ocsigen/tyxml"
+doc: "https://ocsigen.org/tyxml/latest/manual/intro"
+bug-reports: "https://github.com/ocsigen/tyxml/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "ocaml" {>= "4.04"}
+  "tyxml" {= version}
+  "tyxml-syntax" {= version}
+  "alcotest" {with-test}
+  "reason" {with-test}
+  "ppxlib" {>= "0.9"}
```

Comment:
We'll have a lower-bound constraint checker in the CI in a week or two but in the meantime, looking at the code, I believe 0.9 is not high enough. Could try what fits? (try 0.18 first)

Reply:
Ok, I tried, and 0.18 seems to be indeed the minimal version. Thanks a lot.